src/main/webapp/components/NoteGridCard.vue

```diff
@@ -1,36 +1,63 @@
 <template>
-  <div class="note-grid-card" @click="$emit('click')">
-    <div class="thumbnail" :style="thumbnail"></div>
+  <v-col>
+    <v-card class="note-grid-card" @click="$emit('click')"
+            width="208px">
+      <v-img :src="thumbnailSrc"
+             height="180px"
+             position="top">
+        <v-icon>{{ isGNote ? "mdi-google-drive" : "mdi-pdf-box" }}</v-icon>             
+      </v-img>
 
-    <div class="metadata">
-      <div class="note-title">{{title}}</div>
+      <v-divider></v-divider>
 
-      <div class="info-row">
-        <p>{{school}} &bull; {{course}}</p>
+      <v-card-title>
+        <v-list-item-title class="d-block text-truncate">{{title}}</v-list-item-title>
+      </v-card-title>
 
-        <img src="assets/gdrive.webp">
-        {{dateCreated}}
-        <div class="rating-box">
-          {{numFavorites}}
-          <span class="star">&star;</span>          
-        </div>
-      </div>
-    </div>
-  </div>
+      <v-card-subtitle>
+        <v-list-item-subtitle>{{ course }} • {{ school }}</v-list-item-subtitle>
+      </v-card-subtitle>
+
+      <v-card-text>
+        <v-row no-gutters>
+          <v-col>
+            {{dateCreated}}
+          </v-col>
+          <v-col col="1">
+            <v-badge overlap class="ma-1">
+              <template v-slot:badge>{{numFavorites}}</template>
+              <v-icon :color="favorited && 'yellow'">mdi-star</v-icon>
```

Comment:
I tried running this locally, and I got an error saying that the type check failed for prop color because it expected a String but got a Boolean. Maybe you could store a color variable in data and then change it whenever `this.favorited` changes?

Reply:
Ahh good catch, thought I could do something fancy but failed x) made that change.